app.js

```diff
@@ -111,38 +110,18 @@ app.use((req, res, next) => {
 // Set up routes
 require('./src/server/routes')(app);
 
+const NotFoundError = require('./src/server/helpers/error').NotFoundError;
+
 // Catch 404 and forward to error handler
 app.use((req, res, next) => {
-	const err = new Error('Not Found');
-	err.status = status.NOT_FOUND;
-	next(err);
+	next(new NotFoundError());
 });
 
-// Error handlers
-
-/* Development error handler; displays stacktrace to user */
-if (app.get('env') === 'development') {
-	app.use((err, req, res) => {
-		console.log(`Internal Error. Message: ${err.message} Stacktrace...`);
-		console.log(err.stack);
-
-		res.status(err.status || status.INTERNAL_SERVER_ERROR);
-
-		res.render('error', {
-			message: err.message,
-			error: err
-		});
-	});
-}
-
-/* Production error handler; stacktrace is omitted */
-app.use((err, req, res) => {
-	res.status(err.status || status.INTERNAL_SERVER_ERROR);
+const error = require('./src/server/helpers/error');
 
-	res.render('error', {
-		message: err.message,
-		error: {}
-	});
+// Error handler; arity MUST be 4
+app.use((err, req, res, next) => { // eslint-disable-line no-unused-vars
```

Comment:
The arity of the handler has to be four or express doesn't recognize it as an error handler. If the prototype is (err, req, res) then we're actually getting (err=req, req=res, res=next).